performance_regression_test.py

```diff
@@ -107,12 +107,22 @@ def get_test_xml(self, result, test_name=''):
     def generate_stats_json(self, results, cmds=[]):
         metrics = {'test_details': {}, 'setup_details': {}, 'versions': {}, 'results': {}}
 
+        is_gce = False
+        for p in self.params.iteritems():
+            if ('/run/backends/gce', 'cluster_backend', 'gce') == p:
+                is_gce = True
         for p in self.params.iteritems():
```

Comment:
parameter '/run/backends/gce' defines that we run against gce, so we can skip any _aws params from results.
for aws run it could be like '/run/backends/aws/us_east_1'